File: java/experiments/talkfrontend/src/app/model/message/message-create/message-create.ts
export class MessageCreate {
    conversation_idconversation: number;
    user_iduser: number;
    text: string;

    constructor(
        conversation_idconversation: number,
        user_iduser: number,
        text: string
    ) {
        this.conversation_idconversation = conversation_idconversation;
        this.user_iduser = user_iduser;
        this.text = text;
    }

}

File: java/experiments/talk/src/main/java/com/yzumis/talk/model/message/MessageCreate.java
package com.yzumis.talk.model.message;

import java.sql.Timestamp;

public class MessageCreate {

    private Integer conversation_idconversation;
    private Integer user_iduser;
    private String text;
    private Timestamp timestamp;

    public Integer getConversation_idconversation() {
        return conversation_idconversation;
    }

    public Integer getUser_iduser() {
        return user_iduser;
    }

    public String getText() {
        return text;
    }

    public Timestamp getTimestamp() {
        return timestamp;
    }

}
